homeassistant/components/sensor/nut.py

```diff
@@ -217,7 +217,9 @@ def opp_state(self):
             return STATE_TYPES['OFF']
         else:
             try:
-                return STATE_TYPES[self._data.status[KEY_STATUS]]
+                return " ".join(
+                    STATE_TYPES[state]
```

Comment:
Good point! I am also fine with it returning `STATE_UNKNOWN` (since it would then be more likely to be fixed - unless unknown states turns out to be very common in the wild).